bot/exts/moderation/watchchannels/talentpool.py

```diff
@@ -176,13 +175,39 @@ async def nomination_edit_group(self, ctx: Context) -> None:
 
     @nomination_edit_group.command(name='reason')
     @has_any_role(*MODERATION_ROLES)
-    async def edit_reason_command(self, ctx: Context, nomination_id: int, *, reason: str) -> None:
-        """
-        Edits the reason/unnominate reason for the nomination with the given `id` depending on the status.
+    async def edit_reason_command(self, ctx: Context, nomination_id: int, actor: FetchedMember, *, reason: str) -> None:
+        """Edits the reason of `actor` entry for the nomination with the given `id`."""
+        try:
+            nomination = await self.bot.api_client.get(f"{self.api_endpoint}/{nomination_id}")
+        except ResponseCodeError as e:
+            if e.response.status == 404:
+                self.log.trace(f"Nomination API 404: Can't nomination with id {nomination_id}")
```

Comment:
Should be
```suggestion
                self.log.trace(f"Nomination API 404: Can't edit a nomination with id {nomination_id}")
```

This seems to be a typo carried over from the original function. It seems to be in the function below as well, but I can't mark it since it's not a change.